subgraphs/abracadabra/src/mapping.ts

```diff
@@ -193,53 +206,64 @@ export function handleLiquidation(event: LogRepay): void {
   );
 
   liquidateEvent.hash = event.transaction.hash.toHexString();
+  liquidateEvent.nonce = event.transaction.nonce;
   liquidateEvent.logIndex = event.transactionLogIndex.toI32();
-  liquidateEvent.protocol = getOrCreateLendingProtocol().id;
-  liquidateEvent.to = event.params.to.toHexString();
-  liquidateEvent.from = event.params.from.toHexString();
+  liquidateEvent.liquidatee = event.params.to.toHexString();
+  liquidateEvent.liquidator = event.params.from.toHexString();
   liquidateEvent.blockNumber = event.block.number;
   liquidateEvent.timestamp = event.block.timestamp;
   liquidateEvent.market = market.id;
   liquidateEvent.asset = mimToken.id;
   liquidateEvent.amount = collateralAmount;
   liquidateEvent.amountUSD = collateralAmountUSD;
   liquidateEvent.profitUSD = collateralAmountUSD.minus(mimAmountUSD);
-  liquidateEvent.liquidatee = event.params.to.toHexString();
+  liquidateEvent.save();
+
+  let liqudidatedAccount = getOrCreateAccount(liquidateEvent.liquidatee!, event);
+  liqudidatedAccount.liquidateCount = liqudidatedAccount.liquidateCount + 1;
+  liqudidatedAccount.save();
+  addAccountToProtocol("LIQUIDATEE", liqudidatedAccount, event);
```

Comment:
Probably better to make these named constants

Reply:
done, in a new commit.